system-test/search.js

```diff
@@ -67,12 +56,8 @@ describe('Search', function() {
   var INDEX_NAME = generateIndexName();
   var index = search.index(INDEX_NAME);
 
-  before(function(done) {
-    deleteAllDocuments(done);
```

Comment:
Please forgive my lack of familiarity with the search api - if something like a test timeout occurs on a previous Travis run, would there still be lingering documents?